development/includes/blog-column.pug

```diff
@@ -0,0 +1,61 @@
+//- <!-- Blog preview link boxes -->
```

Comment:
Also, please put an underscore in front of all these includes `_blog-column.pug`